tests/core/test_solve.py

```diff
@@ -1144,131 +1146,140 @@ def test_force_reinstall_1():
 
 
 @pytest.mark.integration  # this test is slower, so we'll lump it into integration
-def test_freeze_deps_1():
-    specs = MatchSpec("six=1.7"),
-    with get_solver_2(specs) as solver:
-        final_state_1 = solver.solve_final_state()
-        # PrefixDag(final_state_1, specs).open_url()
-        print([Dist(rec).full_name for rec in final_state_1])
-        order = (
-            'channel-2::openssl-1.0.2l-0',
-            'channel-2::readline-6.2-2',
-            'channel-2::sqlite-3.13.0-0',
-            'channel-2::tk-8.5.18-0',
-            'channel-2::xz-5.2.2-1',
-            'channel-2::zlib-1.2.8-3',
-            'channel-2::python-3.4.5-0',
-            'channel-2::six-1.7.3-py34_0',
-        )
-        assert tuple(final_state_1) == tuple(solver._index[Dist(d)] for d in order)
+def test_freeze_deps_1(pytestconfig):
 
-    # to keep six=1.7 as a requested spec, we have to downgrade python to 2.7
-    specs_to_add = MatchSpec("bokeh"),
-    with get_solver_2(specs_to_add, prefix_records=final_state_1, history_specs=specs) as solver:
-        final_state_2 = solver.solve_final_state()
-        # PrefixDag(final_state_2, specs).open_url()
-        print([Dist(rec).full_name for rec in final_state_2])
-        order = (
-            'channel-2::mkl-2017.0.1-0',
-            'channel-2::openssl-1.0.2l-0',
-            'channel-2::readline-6.2-2',
-            'channel-2::sqlite-3.13.0-0',
-            'channel-2::tk-8.5.18-0',
-            'channel-2::xz-5.2.2-1',
-            'channel-2::yaml-0.1.6-0',
-            'channel-2::zlib-1.2.8-3',
-            'channel-2::python-2.7.13-0',
-            'channel-2::backports-1.0-py27_0',
-            'channel-2::backports_abc-0.5-py27_0',
-            'channel-2::futures-3.1.1-py27_0',
-            'channel-2::markupsafe-0.23-py27_2',
-            'channel-2::numpy-1.13.0-py27_0',
-            'channel-2::pyyaml-3.12-py27_0',
-            'channel-2::requests-2.14.2-py27_0',
-            'channel-2::setuptools-27.2.0-py27_0',
-            'channel-2::six-1.7.3-py27_0',
-            'channel-2::bkcharts-0.2-py27_0',
-            'channel-2::jinja2-2.9.6-py27_0',
-            'channel-2::python-dateutil-2.6.0-py27_0',
-            'channel-2::singledispatch-3.4.0.3-py27_0',
-            'channel-2::ssl_match_hostname-3.4.0.2-py27_1',
-            'channel-2::tornado-4.5.1-py27_0',
-            'channel-2::bokeh-0.12.6-py27_0',
-        )
-        assert tuple(final_state_2) == tuple(solver._index[Dist(d)] for d in order)
+    # https://github.com/pytest-dev/pytest/issues/1599
+    capmanager = pytestconfig.pluginmanager.getplugin('capturemanager')
+    capmanager.suspendcapture()
 
```

Comment:
This whole change is only the above, which sends output to the terminal directly rather than it being captured by pytest, then an indent of the full code of the test inside a `with` context.  None of the bulk of the code has actually changed.